internal/storage/postgres/tree/create_test.go

```diff
@@ -0,0 +1,131 @@
+package tree
+
+import (
+	"context"
+	"github.com/green-ecolution/green-ecolution-backend/internal/entities"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestTreeRepository_Create(t *testing.T) {
+	t.Run("should create a tree with default values", func(t *testing.T) {
+		// given
+		suite.ResetDB(t)
+		suite.InsertSeed(t, "internal/storage/postgres/seed/test/tree")
+		r := NewTreeRepository(suite.Store, mappers)
+
+		// when
+		got, err := r.Create(context.Background())
+
+		// then
+		assert.NoError(t, err)
+		assert.NotNil(t, got)
+		assert.Equal(t, "", got.Species)
+		assert.NotZero(t, got.ID)
+		assert.WithinDuration(t, got.CreatedAt, time.Now(), time.Second)
+		assert.WithinDuration(t, got.UpdatedAt, time.Now(), time.Second)
+		assert.Nil(t, got.TreeCluster)
+		assert.Nil(t, got.Sensor)
```

Comment:
Images and Readonly are missing.

Reply:
I don't think it's necessary to test every attribute. the aim is to show that the function works as expected. And the handling of images isn't part of the create function, so please take a look at the create function to verify this.

Reply:
No the aim is to check, if every data is also returned correctly. It is needed!

Reply:
as discussed.